pylibui/libui/box.py

```diff
@@ -54,13 +54,13 @@ def uiBoxDelete(box, index):
 # - int uiBoxPadded(uiBox *b);
 def uiBoxPadded(box):
     """
-    Sets the padding of the box.
+    Returns whether the box is padded.
 
     :param box: uiBox
     :return: int
     """
 
-    return clibui.uiBoxPadded(box)
+    return bool(clibui.uiBoxPadded(box))
```

Comment:
I'll fix this. But some other widgets like checkbox implemented this in libui so I think I should do so. Maybe we must fix others too.